### ISSUE TITLE
Screen is locked when charging

### ISSUE THREAD

**Issue by DaveDixon** [mt8163/android_device_amazon_karnak_old#29]
HD 8 (2018) rooted.
From TWRP installed lineage-20.0-20230104-UNOFFICIAL-karnak and MindTheGapps-13.0.0-arm-20230808_213154.
Works great on battery, but touch screen doesn't respond when plugged into a power-only micro-USB.

**Comment by 488315** [comment]
> HD 8 (2018) rooted.
> 
> From TWRP installed lineage-20.0-20230104-UNOFFICIAL-karnak and MindTheGapps-13.0.0-arm-20230808_213154.
> 
> Works great on battery, but touch screen doesn't respond when plugged into a power-only micro-USB.
> 
> 
> 
> NOTE: This does not happen when plugged into an actual USB cable, only a charging cable. 

please use the actual charging cable that comes with the device, i never had any issues like this before. it may be issue with your hardware.

**Comment by DaveDixon** [comment]
That cable is long gone. I'll try to find another microUSB cable.

**Comment by DaveDixon** [comment]
tl;dr a new cable fixed the problem. 

It took a while to replace it because it's a remote location and the only power is POE. I had to get a new splitter, and that fixed the problem. But I'm curious about the problem. The old splitter was barely out of spec at 5.27v and less than 1% ripple. Most devices will handle much higher voltages. Indeed, like the Fire 7 that was on that splitter for 4 years, the Ring device that shared it, my Motorola phone and a Galaxy tablet are all fine with it. 

The touch wasn't completely locked out - it's like plugging in the power cable remapped the touch, so that active elements on the screen didn't get the touch, but sometimes other nonactive components would be highlighted, and the status bar at the top and tray at the bottom were activated with most touches. Why would faulty power do that? And why only on this Fire HD 8 with LineageOS? Curious.
